src/ResourceManager/Sql/Commands.Sql/ImportExport/Cmdlet/GetAzureSqlDatabaseImportExportStatus.cs

```diff
@@ -0,0 +1,84 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Azure.Commands.Sql.ImportExport.Model;
+using Microsoft.Azure.Commands.Sql.ImportExport.Service;
+
+namespace Microsoft.Azure.Commands.Sql.ImportExport.Cmdlet
+{
+    /// <summary>
+    /// Defines the AzureRmSqlDatabaseImportExportStatus cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseImportExportStatus", ConfirmImpact = ConfirmImpact.Low)]
```

Comment:
Why ConfirmImpact.Low?  Using ConfirmImpact.Medium avoids prompts.  Also, you don't want this setting without SupportsShouldProcess=true